### PR TITLE
suggested extensions

### PR DESCRIPTION
**Pull request by ConradIrwin**
Follow-up from #9138

Release Notes:

- Adds suggested extensions for some filetypes ([#7096](https://github.com/zed-industries/zed/issues/7096)).

### CODE REVIEW

File: crates/extension/src/extension_store.rs [hunk 1]
```diff
@@ -461,6 +447,49 @@ impl ExtensionStore {
...
+    pub fn install_latest_extension(
+        &mut self,
+        extension_id: Arc<str>,
+        cx: &mut ModelContext<Self>,
+    ) {
+        log::info!("installing extension {extension_id} latest version");
+
+        let url = self
+            .http_client
+            .build_zed_api_url(&format!("/extensions/{extension_id}/0.0.2/download"));
```
[the remark this answers is not in the document]
Comment (ConradIrwin) [reply crates/extension/src/extension_store.rs]: Thanks! I needed that for testing as I still haven't figured out how to test extensions locally.

File: crates/workspace/src/notifications.rs
Comment (Feel-ix-343) [inline crates/workspace/src/notifications.rs:1]: @ConradIrwin I am pretty sure these changes should be removed. As of I conversation I don't think they are used

Comment (Feel-ix-343) [reply crates/workspace/src/notifications.rs:1]: Oh never mind. I see you changed the suggest notification behavior. I do remember there being an issue with overflow when the click message is too long, however. I think overflow_hidden (or akin) should be set. The extension names probably don't trigger this, but obviously this is quite a general API.

**Comment by Feel-ix-343** [comment]
@ConradIrwin shall we open new PRs to suggest new extensions? (I really want to add my [markdown language server](https://github.com/Feel-ix-343/markdown-oxide)! I published it as an extension yesterday `markdown-oxide`)

**Comment by Feel-ix-343** [comment]
Yes fair I agree

**Pull request merged**